chain/client/src/sync/sync_actor.rs

```diff
@@ -0,0 +1,43 @@
+use near_async::messaging::Sender;
+use near_primitives::hash::CryptoHash;
+use near_store::ShardUId;
+use tracing::log::info;
+
+use super::adapter::SyncMessageForClient;
+use near_network::state_sync::SyncActorMessageForNetwork;
+struct MessageSenders {
+    client_adapter: Sender<SyncMessageForClient>,
```

Comment:
nit: feel free to leave as is, but these two fields could just go directly in `struct SyncActor` right? seems simpler to me that way but it's a style preference, do it as you like it

Reply:
I would keep them bundled.